ui/apps/dashboard/src/components/Environments/Environments.tsx

```diff
@@ -0,0 +1,162 @@
+import { type Route } from 'next';
+import Link from 'next/link';
+import { Button } from '@inngest/components/Button';
+import { Link as InngestLink } from '@inngest/components/Link';
+import { RiErrorWarningLine } from '@remixicon/react';
+
+import Toaster from '@/components/Toaster';
+import { EnvironmentType, LEGACY_TEST_MODE_NAME, type Environment } from '@/utils/environments';
+import { EnvironmentArchiveButton } from './old/EnvironmentArchiveButton';
+import EnvironmentListTable from './old/EnvironmentListTable';
+
+export default async function Environments({ envs }: { envs: Environment[] }) {
+  // Break the environments into different groups
+  const legacyTestMode = envs.find(
+    (env) => env.type === EnvironmentType.Test && env.name === LEGACY_TEST_MODE_NAME
+  );
+  const branchParent = envs.find((env) => env.type === EnvironmentType.BranchParent);
+  const branches = envs.filter((env) => env.type === EnvironmentType.BranchChild);
+  const otherTestEnvs = envs.filter(
+    (env) => env.type === EnvironmentType.Test && env.name !== LEGACY_TEST_MODE_NAME
+  );
+
+  return (
```

Comment:
The old environments page had it's own copy of the application nav embedded in it. This new is one is just a copy with that removed since that have is now external.